public/index.html.template

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>${description}</title>
+
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+
+  <style>
+    h3 {
+      margin-left: 25px;
+      text-align: center;
+    }
+
+    a, a:visited {
+      color: #3f5dff;
+    }
+
+    h3 a {
+      margin-left: 10px;
+    }
+
+    a:hover, a:focus, a:active {
+      color: #001956;
+    }
+
+    .power {
+      position: absolute;
+      bottom: 25px;
+      left: 50%;
+      transform: translateX(-50%);
+    }
+  </style>
+</head>
+
+<body>
+  <h1 style="text-align: center">${name}@${version}</h1>
+  <h3>OpenAPI spec: <a href="/openapi.json">/openapi.json</a></h4>
+  <h3>API Explorer: <a href="/explorer">/explorer</a></h4>
```

Comment:
IIRC, applications can customize the URL of both OpenAPI spec and Explorer. See https://github.com/strongloop/loopback-next/pull/1637, [Customize How OpenAPI Spec is Served](https://github.com/strongloop/loopback-next/blob/eca79a45244c3e9215face9b9a35b3d3d2e5cb0a/docs/site/Server.md#customize-how-openapi-spec-is-served) and [Configure the API Explorer](https://github.com/strongloop/loopback-next/blob/eca79a45244c3e9215face9b9a35b3d3d2e5cb0a/docs/site/Server.md#configure-the-api-explorer).

IMOL, the home page should respect these settings. Eventually :) I am fine to keep the current static version as the first baby step.